kratos/includes/node.h

```diff
@@ -156,11 +153,10 @@ class Node : public Point,  public IndexedObject, public Flags
     /// 1d constructor.
     Node(IndexType NewId, double const& NewX)
         : BaseType(NewX)
-        , IndexedObject(NewId)
         , Flags()
+        , mNodalData(NewId)
         , mDofs()
         , mData()
-        , mSolutionStepsNodalData()
         , mInitialPosition(NewX)
 #ifdef _OPENMP
         , mNodeLock()

```

Comment:
just to tell that i think we could emulate the lock by using a bool and c++11 functions